src/meta/src/manager/catalog/mod.rs

```diff
@@ -1295,7 +1295,7 @@ impl CatalogManager {
         table_id: TableId,
         internal_table_ids: Vec<TableId>,
     ) -> MetaResult<bool> {
-        let (table, internal_tables) = {
+        let table = {
```

Comment:
Why do we need to call `self.core.lock()` two times separately in `cancel_create_materialized_view_procedure`?

Reply:
Hmm not sure either. I just tried unifying it in one block let's see if the tests pass.